src/components/AllPagesPanel/AllPagesPanel.tsx

```diff
@@ -72,15 +75,28 @@ export const AllPagesPanel: React.FC<AllPagesPanelProps> = (props) => {
         [onMenuItemsChanged, editMenuProps],
     );
 
+    const onDragEnd = useCallback(() => {
+        setDragingItemTitle(null);
+    }, [onMenuItemsChanged]);
```

Comment:
What is the purpose of this dependency if there is no reference to it in the callback code?